services/ui-src/src/forms/mcpar/mcpar.json

```diff
@@ -179,7 +179,7 @@
             "id": "absse",
             "fields": [
               {
-                "id": "absse-a8",
+                "id": "bssEntityName",
```

Comment:
Each report will have lots of different BSS entities added and this field will be pulled into entity pages, so:
```suggestion
                "id": "bssEntities",
```